packages/shared/components/inputs/RecipientInput.svelte

```diff
@@ -3,12 +3,13 @@
     import { WalletAccount } from '@lib/typings/walletAccount'
     import { ADDRESS_LENGTH, validateBech32Address } from '@lib/utils'
     import { RecipientAccountSelector, TextInput, InputContainer, Modal } from 'shared/components'
+    import { activeProfile } from '@lib/profile'
+    import { getNetwork } from '@core/network'
 
     export let recipient: string | WalletAccount
     export let disabled = false
 
-    // TODO: get ADDRESS_PREFIX from profile network info
-    const ADDRESS_PREFIX = 'atoi'
+    const addressPrefix = getNetwork($activeProfile.networkProtocol, $activeProfile.networkType)?.bech32Hrp
```

Comment:
we should consider if this is a private network and if so get this from the nodeInfo store with something like $nodeInfo?.protocol?.bech32HRP